tests/unit/test-extension-runners/test.chromium.js

```diff
@@ -49,7 +49,7 @@ describe('util/extension-runners/chromium', async () => {
   it('uses the expected chrome flags', () => {
     // Flags from chrome-launcher v0.14.0
     const expectedFlags = [
-      '--disable-features=Translate',
+      '--disable-features=Translate,OptimizationHints,MediaRouter,DialMediaRouteProvider,CalculateNativeWinOcclusion,InterestFeedContentSuggestions,CertificateTransparencyComponentUpdater,AutofillServerCommunication,PrivacySandboxSettings4',
```

Comment:
Do any of these cover the new search engines picker? It's quite annoying.

Starting a new profile (`web-ext run --target chromium`) is basically a whack-a-mole.